tests/e2e/e2e_test.go

```diff
@@ -1108,8 +1130,11 @@ func (s *IntegrationTestSuite) TestPacketForwarding() {
 		if err != nil {
 			return false
 		}
-		count := response["count"].(float64)
-		return err == nil && count == 0
+		countValue, ok := response["count"].(float64)
+		if !ok {
+			return false
+		}
+		return err == nil && countValue == 0
```

Comment:
I think there is no need to check for error being nil, because at the point of return it has already been checked (line 1130). 

```suggestion
		return countValue == 0
```